javascript-light-bulb/main.js

```diff
@@ -0,0 +1,16 @@
+var $background = document.querySelector('.background');
+var $bulb = document.querySelector('.bulb');
+var isOn = true;
+
+$bulb.addEventListener('click', onAndOff);
+
+function onAndOff(event) {
+  if (isOn) {
```

Comment:
So if the light `isOn`, we turn on the light? That sounds backwards. Please reverse these two conditional blocks